Hyper/PlayerData/InventorySystem/Items/Lamp.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Common.Meshes;
+using OpenTK.Mathematics;
+using Physics.TypingUtils;
+
+namespace Hyper.PlayerData.InventorySystem.Items;
+internal class Lamp : Item
+{
+    public override string Id => "lamp";
+
+    public override bool IsStackable => true;
+
+    public override void Use(Scene scene)
+    {
+        if (!scene.Player.Inventory.TryRemoveItem("lamp"))
+            return;
+        CreateLamp(scene);
+    }
+
+    private static void CreateLamp(Scene scene)
+    {
+        scene.LightSources.Add(new Common.Meshes.Lamp(CubeMesh.Vertices,
+                    position: new Vector3(Conversions.ToOpenTKVector(scene.Player.RayOrigin)),
```

Comment:
remove one tab

Reply:
also I think the default values of the lamp should be in lamp class. Here it should just be Lemp.CreateNew(position) or something like that